vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -192,9 +191,11 @@ possible with the built-in functions then you can create a [custom figure](custo
             ),
         ]
 
+        # Create a layout with four rows and columns. The KPI cards are positioned in the first eight cells, while the remaining cells are empty.
```

Comment:
Similar comment - would put it directly above the Layout model call as this is supposed to be the explanation for the cryptic grid specification